gradle.properties

```diff
@@ -4,8 +4,9 @@ versionSuffix=SNAPSHOT
 kotlin.code.style=official
 
 infraVersion = 0.3.0-dev-64
-kotlinVersion = 1.6.0
+kotlinVersion = 1.7.0
```

Comment:
Then, users with previous Kotlin versions won't be able to use this version of the library.
In particular, users with 1.6.20 are affected by the problem and won't get this fix.